app/assets/stylesheets/application.scss

```diff
@@ -108,11 +108,11 @@ a.config_group_name {
 }
 
 .editable {
-  background: url("edit.png") no-repeat scroll 98% 6px transparent;
+  background: image-url("edit.png") no-repeat scroll 98% 6px transparent;
```

Comment:
All of these changes make the url fetch the digested asset (edit.png3820910893120893) instead of the undigested asset. Rails 4 does not serve undigested assets anymore.